src/components/PrimaryPanes/Sources.css

```diff
@@ -62,6 +62,7 @@
   display: flex;
   flex: 1;
   height: 100%;
+  padding-bottom: 4px;
```

Comment:
i'm not sure why we have this one in addition to the bottom padding we have on the tree as well.
Here it looks like we have a 4px padding at the top, then a 8px one at the bottom, which is a bit odd.

Reply:
I removed this padding. Thanks.